src/blocks/Author/__tests__/Author.visual.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import {test} from '../../../../playwright/core/index';
+
+import {Default} from './helpers';
+
+test.describe('Author', () => {
+    test('render stories <Default>', async ({mount, expectScreenshot, defaultDelay}) => {
+        await mount(<Default />);
+        await defaultDelay();
```

Comment:
await defaultDelay(); is it really necessary?

Reply:
I think it is necessary